src/main/java/org/checkerframework/specimin/UnsolvedSymbolVisitor.java

```diff
@@ -1588,6 +1588,9 @@ public boolean updatedAddedTargetFilesForPotentialEnum(FieldAccessExpr expr) {
       return false;
     }
     if (resolved.isEnumConstant()) {
+      if (JavaLangUtils.inJdkPackage(resolved.getType().describe())) {
+        return true;
```

Comment:
I'm surprised this is `return true` rather than `return false`, since in this case no update actually occurs.

Reply:
updated, does not seem to change functionality

Reply:
probably means we should have another test case, but this seems fine to me